Game/enemies.py

```diff
@@ -0,0 +1,57 @@
+import random
+
+class NoEnemy(object):
+
+	def __init__(self, weapon, armor):
+		self.type = "none"
+		self.hitpoints = random.randint(0, 0)
+		self.weapon = weapon
+		self.armor = armor
+
+class Guard(object):
```

Comment:
I'd definitely create a generic class and inherit everything from it since they're identical basically besides hitpoints.